src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -111,6 +111,10 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
   }
 
   public setConfig(config: EntitiesCardConfig): void {
+    if (!config || !config.entities.length) {
+      throw new Error("Entities must be specified");
+    }
+
```

Comment:
This was done on purpose so the card didn't error the UI editor. I would make sure we arent making new users see red errors when opening up the card picker and choosing a card

Reply:
They should not see any error if they add a new card, since the editor automatically fills in entities. Only if they manually delete all those entries in the editor without adding new ones will they (at least temporarily) see the red box. So I think we are not impacting new users.